schema/codelists/relatedProcess.csv

```diff
@@ -1,9 +1,10 @@
 Code,Title,Description
-framework,Framework agreement procedure first stage,This contracting process follows on from the related process to establish a framework. The suppliers on the framework are listed in the award section of the related process.
+framework,Framework agreement procedure first stage,This contracting process follows on from the related process that established a framework agreement. The suppliers on the framework are listed in the award section of the related process.
 planning,Planning process,This contracting process follows on from the related planning process.
 parent,Parent contract (for sub-contracts),This contracting process might result in a sub-contract of the related process.
-prior,Prior process,This contracting process is the renewal or replacement of the related prior process.
-unsuccessfulProcess,Unsuccessful process,This contracting process follows on from an previous unsuccessful process.
-subContract,Sub-contract,The related process might result in a sub-contract of this contract.
-replacementProcess,Replacement process,The related process might result in the replacement of this contract.
-renewalProcess,Renewal process,The related process might result in the renewal of this contract.
+prior,Prior process,This contracting process follows on from the related successful process.
```

Comment:
I think we want a better name than "prior process" for this and sub-codes. At minimum, "Successful process". The subcodes don't need to follow the format "Parent code (specifier)". They can be "Additional goods, services or works", etc. (whatever seems clearest and most consistent).

Reply:
Noting that we probably need to clarify the semantics per https://github.com/open-contracting/standard/issues/1520#issuecomment-1762330688 before deciding on names.

Reply:
Deprecated 'prior' and replaced it with 'suucessfulProcess' in https://github.com/open-contracting/standard/pull/1645/commits/c114f7d58731751bcb6118426fd3b0e430a299ef

Reply:
Ah, I don't think we want to change the actual code, as it is in use. We can just change the title (I said name before).

Reply:
Done in 1f4ec0786fd9a6978f0563a20bd11434c9f159df